src/components/NavBar/NavBar.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { Layout, Menu } from 'antd';
+import { Link } from 'react-router-dom';
+
+import {
+  ClockCircleFilled,
+  HomeFilled,
+  SafetyCertificateFilled,
+  DatabaseFilled,
+  BellFilled,
+  GlobalOutlined,
+} from '@ant-design/icons';
+
+const { Sider } = Layout;
+
+const VerticalMenu = () => {
+  return (
+    <Sider width={200}>
```

Comment:
if the width is hardcoded it should be defined as a constant such as
```JavaScript
const SIDER_WIDTH = 200;
```
after the imports in this file